pkg/gds/admin/v2/client.go

```diff
@@ -224,6 +224,21 @@ func (s APIv2) Resend(ctx context.Context, in *ResendRequest) (out *ResendReply,
 	return out, nil
 }
 
+func (s APIv2) Timeline(ctx context.Context) (out *TimelineReply, err error) {
```

Comment:
Not sure if here we want `(s *APIv2)` instead? I remember I had a similar comment from one of the localew reviews, that using a pointer would save us some space.

Reply:
Good question, I was trying to follow the existing format for the other function definitions in the file but it looks like @bbengfort is adding the * in his new [PR](https://github.com/trisacrypto/directory/pull/129#discussion_r720226494) anyway.

Reply:
In a previous PR I did have to update the receiver to a pointer. You can think about the `(s APIv2)` as similar to `self` in Python, it's the "instance" (no classes in Go, but the metaphor is good) referred to by the method. If you have a pointer to the instance, you can modify the instance; without the pointer, you have a copy passed by value, which means no modifications will take effect.

See the go example here: https://play.golang.org/p/v35S-ZNbiSX